package.json

```diff
@@ -5,6 +5,8 @@
   "scripts": {
     "start": "node -r dotenv/config ./bin/www --env=production",
     "dev": "DEBUG=* nodemon -r dotenv/config ./bin/www",
+    "test": "node -r dotenv/config ./node_modules/.bin/cypress run --env ENVIRONMENT=test",
+    "test:cli": "node -r dotenv/config ./node_modules/.bin/cypress open --env ENVIRONMENT=test",
```

Comment:
I think `test:interactive` or something to that effect would be more appropriate here than `test:cli`.